src/Raven.Server/Documents/Handlers/Debugging/ThreadsHandler.cs

```diff
@@ -18,6 +19,9 @@ public class ThreadsHandler : RequestHandler
         [RavenAction("/admin/debug/threads/stack-trace", "GET", AuthorizationStatus.Operator)]
         public Task StackTrace()
         {
+            if (PlatformDetails.RunningOnPosix)
+                throw new NotSupportedException("Stack Traces Capture is not supported on linux.");
```

Comment:
Maybe "not supported on POSIX" since it applies to Linux but macOS as well

Reply:
done